Dockerfile

```diff
@@ -110,33 +110,35 @@ RUN set -eux \
   && apk update --no-cache \
   && apk upgrade --no-cache \
   && apk add --no-cache bash \
-  && apk add --no-cache curl \
-  && apk add --no-cache docker \
-  && apk add --no-cache git \
-  && apk add --no-cache jq \
-  && apk add --no-cache make \
-  && apk add --no-cache ncurses \
-  && apk add --no-cache openssh \
-  && apk add --no-cache openssl \
-  && apk add --no-cache python3 \
-  && apk add --no-cache zip \
-  && if [ ! -e /usr/bin/python ]; then ln -sf python3 /usr/bin/python ; fi \
-  && python3 -m ensurepip \
-  && rm -r /usr/lib/python*/ensurepip \
-  && pip3 install --no-cache --upgrade pip setuptools wheel \
-  && if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi \
-  && python3 -m pip install ply --no-cache-dir \
-  && python3 -m pip install pyhcl --no-cache-dir \
-  && python3 -m pip install requests --no-cache-dir \
-  && python3 -m pip install slack_sdk --no-cache-dir \
-  && if [ "${AWS}" == "yes" ]; then python3 -m pip install boto3 --no-cache-dir; python3 -m pip install awscli --no-cache-dir; fi \
+	curl \
+	docker \
+	git \
+	jq \
+	make \
+	ncurses \
+	openssh \
+	openssl \
+	python3 \
+	zip \
+  ; if [ ! -e /usr/bin/python ]; then ln -sf python3 /usr/bin/python ; fi \
+  ; python3 -m ensurepip \
+  ; rm -r /usr/lib/python*/ensurepip \
+  ; pip3 install --no-cache --upgrade pip setuptools wheel \
+  ; if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi \
+  ; python3 -m pip install --no-cache-dir ply \
+  	pyhcl \
+	requests \
+	slack_sdk \
+  ; if [ "${AWS}" == "yes" ]; then python3 -m pip install boto3 --no-cache-dir; python3 -m pip install awscli --no-cache-dir; fi \
 #  && if [ "${GCP}" == "yes" ]; then echo GCP; fi \
 #  && if [ "${AZURE}" == "yes" ]; then echo AZURE; fi \
-  && mkdir -m 700 /root/.ssh \
-  && touch -m 600 /root/.ssh/known_hosts \
-  && ssh-keyscan -t rsa github.com > /root/.ssh/known_hosts \
-  && rm -rf /var/cache/* \
-  && rm -rf /root/.cache/*
+  ; printf '%s\n' '[url "https://github.com/"]' \
```

Comment:
I'd still suggest you giving something like this a try since ssh access to github.com is really not needed!

Reply:
Actually I need it very much. To disable SSH I need to change a lot of CI/CD processes in the company, right now it's far in my backlog.

Right now I'm happy I've revisited this code after so long, thanks to you :)

Reply:
How do you use the ssh-key of github except over git? This configuration will get rid of any tool that accesses github through git (except you are authenticating towards github.com through ssh of course sry)

Reply:
Worker nodes have private SSH keys and I run Docker with `--volume ~/.ssh/id_rsa_github_integration:/root/.ssh/id_rsa --volume ~/.gitconfig:/root/.gitconfig` to access private repositories in organization.